lib/screens/find_pw.dart

```diff
@@ -38,7 +36,7 @@ class _FindPwState extends State<FindPw> {
 
     try {
       await _auth.sendPasswordResetEmail(email: _email);
-      General.instance.showSingleDialog(
+      navigation.showSingleDialog(
```

Comment:
It is safe to check if the context is mounted after `await`.

Please note this for the next time.